aws/main.tf

```diff
@@ -535,3 +603,18 @@ module "alb_eth_bytecode_db" {
   security_groups   = module.lb_microservices_sg.security_group_id
   tags              = local.final_tags
 }
+
+module "alb_xchain" {
+  count               = var.xchain_settings["enabled"] ? 1 : 0
+  source              = "./alb"
+  name                = "${var.vpc_name != "" ? var.vpc_name : ""}-xchain"
+  internal            = false
+  vpc_id              = local.vpc_id_rule
+  subnets             = local.subnets_rule
+  backend_port        = 4000
+  health_check_path   = "/healthz"
+  name_prefix         = "xchan-"
```

Comment:
```suggestion
  name_prefix         = "xchain-"
```

Reply:
can't do that, there's a 6char limit haha very weird

Reply:
how about `xchn-`

Reply:
yea we could probably do that. updating